workflow-templates/phpunit-oci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
 
     services:
       oracle:
-        image: deepdiver/docker-oracle-xe-11g # 'wnameless/oracle-xe-11g-r2'
+        image: ghcr.io/nextcloud/continuous-integration-oracle-11g:latest
```

Comment:
The process control (PCNTL) extensions are required in case you want to interrupt long running commands - see https://www.php.net/manual/en/book.pcntl.php
Additionally the function 'pcntl_signal' and 'pcntl_signal_dispatch' need to be enabled in your php.ini.
Error while trying to initialise the database: Failed to connect to the database: An exception occurred in the driver: ORA-12541: TNS:no listener

Reply:
https://github.com/nextcloud/spreed/actions/runs/5142502055/jobs/9256324035?pr=9666

I have no clue...